org.osgi.test.junit5.cm/src/test/java/org/osgi/test/junit5/cm/ConfigAnnotationTest.java

```diff
@@ -91,51 +83,51 @@ public void testFieldConfiguration() throws Exception {
 	@WithConfiguration(pid = PARAM_PID, properties = {
 		@ConfigEntry(key = "bar", value = "foo")
 	})
-	public void testParameterConfiguration(
-		@InjectConfiguration(PARAM_PID) Configuration configuration)
+	public void testParameterConfiguration(@InjectConfiguration(PARAM_PID) Configuration configuration)
 		throws Exception {
 
 		Configuration cs = ConfigUtil.getConfigsByServicePid(ca, PARAM_PID);
 		assertThat(cs).isEqualTo(configuration);
 
-		assertNull(configuration.getProperties()
-			.get("foo"));
+		assertThat(Dictionaries.asMap(cs.getProperties()))
+			.containsExactlyEntriesOf(Dictionaries.asMap(configuration.getProperties()));
```

Comment:
I just updated DictionaryAssert to include method variants which take Dictionary arguments. So you can now say:

```java
assertThat(cs.getProperties()).containsExactlyInAnyOrderEntriesOf(configuration.getProperties());
```

Reply:
i have seen the pr. that helps!